agent/util-scripts/pbench-move-results

```diff
@@ -8,8 +8,24 @@ pbench_bin="$(realpath -e ${script_path}/..)"
 . "${pbench_bin}"/base
 
 function usage() {
-    printf "usage:\n"
-    printf "${script_name} [--help] [--controller=<controller>] [--user=<user>] [--prefix=<path>] [--xz-single-threaded] [--show-server]\n"
+    printf -- "usage:\n%s [--help] --user=<user> [--controller=<controller>]" "${script_name}"
+    printf -- " [--prefix=<path>] [--xz-single-threaded] [--show-server]\n"
+    printf -- "\nThe '--user' option value is required, and can also be provided\n"
+    printf -- "by the 'PBENCH_USER' environment variable.\n"
+    printf -- "\nThe '--controller' option is also required, but defaults to\n"
+    printf -- "the 'hostname -f' value, or the 'PBENCH_CONTROLLER' environment\n"
+    printf -- "variable.\n"
```

Comment:
You cannot have a switch which is required but which provides a default!  😀

"The '--controller' option may be used to override the value provided by the 'PBENCH_CONTROLLER' environment variable; if neither value is available, the result of 'hostname -f' is used.  (If no value is available, the command will exit with an error.)"